### PR TITLE
Doc updates related to .NET Standard, .NET Core, and SDK-style projects

### PR DESCRIPTION
**Pull request by Mikejo5000**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[0dd5bdb](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1454/commits/0dd5bdbae5a35f0d7fc3960f4681eb0ad136e055)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Creating-a-Package.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/create-packages/Creating-a-Package.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/creating-a-package?branch=pr-en-us-1454) | [Details](#user-content-8bc9441c-d8f0f27c)
[docs/reference/migrate-packages-config-to-package-reference.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/reference/migrate-packages-config-to-package-reference.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference?branch=pr-en-us-1454) | [Details](#user-content-3aa78c4d-d8f0f27c)
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/reference/nuspec.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1454) | [Details](#user-content-8224f5a0-d8f0f27c)
[docs/consume-packages/Overview-and-Workflow.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/consume-packages/Overview-and-Workflow.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/overview-and-workflow?branch=pr-en-us-1454) |
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1454) |
[docs/install-nuget-client-tools.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/install-nuget-client-tools?branch=pr-en-us-1454) |
[docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli?branch=pr-en-us-1454) |
[docs/quickstart/create-and-publish-a-package-using-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/create-and-publish-a-package-using-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?branch=pr-en-us-1454) |
[docs/quickstart/install-and-use-a-package-in-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/install-and-use-a-package-in-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio?branch=pr-en-us-1454) |
[docs/what-is-nuget.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/what-is-nuget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/what-is-nuget?branch=pr-en-us-1454) |

<a id="8bc9441c-d8f0f27c"></a>
### [docs/create-packages/Creating-a-Package.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/create-packages/Creating-a-Package.md)
 - **Line 18**: **[Warning]** `````Invalid file link:(~/reference/migrate-packages-config-to-package-reference#create-a-package-after-migration.md).`````

<a id="3aa78c4d-d8f0f27c"></a>
### [docs/reference/migrate-packages-config-to-package-reference.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/reference/migrate-packages-config-to-package-reference.md)
 - **Line 88**: **[Warning]** `````Invalid file link:(~/reference/msbuild-targets#packing-using-a-nuspec).`````

<a id="8224f5a0-d8f0f27c"></a>
### [docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/reference/nuspec.md)
 - **Line 32**: **[Warning]** `````Invalid file link:(~/reference/msbuild-targets#pack-target).`````
 - **Line 32**: **[Warning]** `````Invalid file link:(~/reference/msbuild-targets#packing-using-a-nuspec.md).`````
 - **Line 34**: **[Warning]** `````Invalid file link:(~/reference/migrate-packages-config-to-package-reference#create-a-package-after-migration.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201905242127572238-1454%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gDRlIFZnY1swGaZFYLp%2BUtIIm6cxk8ernMPSZmHx13g%3D&st=2019-05-24T21%3A24%3A04Z&se=2019-06-24T21%3A29%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[40c1a65](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1454/commits/40c1a6510a1839b73f4e74d4b5f4bed30b505283)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Overview-and-Workflow.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/consume-packages/Overview-and-Workflow.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/overview-and-workflow?branch=pr-en-us-1454) |
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1454) |
[docs/create-packages/Creating-a-Package.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/create-packages/Creating-a-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/creating-a-package?branch=pr-en-us-1454) |
[docs/install-nuget-client-tools.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/install-nuget-client-tools?branch=pr-en-us-1454) |
[docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli?branch=pr-en-us-1454) |
[docs/quickstart/create-and-publish-a-package-using-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/create-and-publish-a-package-using-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?branch=pr-en-us-1454) |
[docs/quickstart/install-and-use-a-package-in-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/install-and-use-a-package-in-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio?branch=pr-en-us-1454) |
[docs/reference/migrate-packages-config-to-package-reference.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/reference/migrate-packages-config-to-package-reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference?branch=pr-en-us-1454) |
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/reference/nuspec.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1454) |
[docs/what-is-nuget.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/what-is-nuget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/what-is-nuget?branch=pr-en-us-1454) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201905242146343356-1454%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2BNOGueQYvHEtkWWnBD0zvA8kDXWYxpMAP7SjHwRU8Oo%3D&st=2019-05-24T21%3A43%3A01Z&se=2019-06-24T21%3A48%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[da008a9](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1454/commits/da008a9301e25795b12dd06470523908eeb9c857)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Overview-and-Workflow.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/consume-packages/Overview-and-Workflow.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/overview-and-workflow?branch=pr-en-us-1454) |
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1454) |
[docs/create-packages/Creating-a-Package.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/create-packages/Creating-a-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/creating-a-package?branch=pr-en-us-1454) |
[docs/install-nuget-client-tools.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/install-nuget-client-tools?branch=pr-en-us-1454) |
[docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli?branch=pr-en-us-1454) |
[docs/quickstart/create-and-publish-a-package-using-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/create-and-publish-a-package-using-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?branch=pr-en-us-1454) |
[docs/quickstart/install-and-use-a-package-in-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/install-and-use-a-package-in-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio?branch=pr-en-us-1454) |
[docs/reference/migrate-packages-config-to-package-reference.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/reference/migrate-packages-config-to-package-reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference?branch=pr-en-us-1454) |
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/reference/nuspec.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1454) |
[docs/what-is-nuget.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/what-is-nuget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/what-is-nuget?branch=pr-en-us-1454) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201905242200264870-1454%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gYEJkpbcpwZzNPF%2BwJOEV9VfVShERZCHiT%2Bl2TpIDwE%3D&st=2019-05-24T21%3A56%3A58Z&se=2019-06-24T22%3A01%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/install-nuget-client-tools.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ The [MSBuild CLI](reference/msbuild-targets.md) also provides the ability to res
 
 The two NuGet CLI tools are `dotnet.exe` and `nuget.exe`. See [feature availability](#feature-availability) for a comparison.
 
+* To target .NET Core or .NET Standard, use the dotnet CLI. The dotnet CLI is required for the SDK-style project format, which uses the [SDK attribute](/dotnet/core/tools/csproj#additions).
```
Comment (nkolev92) [inline docs/install-nuget-client-tools.md]: Maybe we can mention SDK based projects again?
dotnet CLI and SDK projects can target .NET Framework.

File: docs/install-nuget-client-tools.md [hunk 2]
```diff
@@ -59,7 +62,7 @@ Installation:
 - Visual Studio on Windows: The **NuGet Package Manager** is included with Visual Studio 2012 and later. The Package Manager provides the [Package Manager UI](tools/package-manager-ui.md) and the [Package Manager Console](tools/package-manager-console.md), through which you can run most NuGet operations.
   - The Visual Studio 2017 installer includes the NuGet Package Manager with any workload that employs .NET. To install separately, or to verify that the Package Manager is installed, run the Visual Studio 2017 installer and check the option under **Individual Components > Code tools > NuGet package manager**.
   - The Package Manager UI and Console are unique to Visual Studio on Windows. They are not presently available on Visual Studio for Mac.
-  - Visual Studio does not automatically include the `nuget.exe` CLI, which must be installed separately as described earlier.
+  - Visual Studio does not automatically include a CLI tool, which is required to support NuGet features in the IDE. You can use either the `dotnet` CLI or the the `nuget.exe` CLI, which must be installed separately as described earlier.
```
Comment (nkolev92) [inline docs/install-nuget-client-tools.md]: The dotnet SDK is installed automatically in some workloads such as the .NET Core workload. 
Not sure how we want to express that, if at all.

File: docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: Creating and publishing a NuGet package using the dotnet CLI
 description: A walkthrough tutorial on creating and publishing a NuGet package using the .NET Core CLI, dotnet.
 author: karann-msft
 ms.author: karann
-ms.date: 01/24/2018
+ms.date: 05/24/2019
```
Comment (nkolev92) [inline docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md]: There are no changes here.

File: docs/reference/nuspec.md [hunk 4]
```diff
@@ -23,6 +23,16 @@ In this topic:
 - [Including content files](#including-content-files)
 - [Example nuspec files](#example-nuspec-files)
 
+## Project type compatibility
```
Comment (nkolev92) [inline docs/reference/nuspec.md]: 🥇

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[f72fe9b](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1454/commits/f72fe9b27fb12d02a2ddc19d053434a1dcd1404a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Overview-and-Workflow.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/consume-packages/Overview-and-Workflow.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/overview-and-workflow?branch=pr-en-us-1454) |
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1454) |
[docs/create-packages/Creating-a-Package.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/create-packages/Creating-a-Package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/creating-a-package?branch=pr-en-us-1454) |
[docs/install-nuget-client-tools.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/install-nuget-client-tools?branch=pr-en-us-1454) |
[docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/create-and-publish-a-package-using-the-dotnet-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli?branch=pr-en-us-1454) |
[docs/quickstart/create-and-publish-a-package-using-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/create-and-publish-a-package-using-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?branch=pr-en-us-1454) |
[docs/quickstart/install-and-use-a-package-in-visual-studio.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/quickstart/install-and-use-a-package-in-visual-studio.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio?branch=pr-en-us-1454) |
[docs/reference/migrate-packages-config-to-package-reference.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/reference/migrate-packages-config-to-package-reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference?branch=pr-en-us-1454) |
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/reference/nuspec.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1454) |
[docs/what-is-nuget.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-updates/docs/what-is-nuget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/what-is-nuget?branch=pr-en-us-1454) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C29%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201905291943492287-1454%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=PcGAUZ2GsyzU2PXgaLm%2F5objUmD1PN3TJl0XPCSE%2FlM%3D&st=2019-05-29T19%3A39%3A48Z&se=2019-06-29T19%3A44%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Mikejo5000** [comment]
Thanks, @nkolev92 ! I'll merge it.

**Pull request merged**